client/src/utils/buildCalendar.js

```diff
@@ -0,0 +1,21 @@
+export default function buildCalendar(value) {
+  const startDay = value.clone().startOf("month").startOf("week");
+  const endDay = value.clone().endOf("month").endOf("week");
+  const day = startDay.clone().subtract(1, "day");
+  const calendar = [];
+
+  while (day.isBefore(endDay, "day")) {
+    calendar.push(
+      Array(7)
+        .fill(0)
+        .map(() => {
+					const tempDay = day.add(1, "day").clone();
+          return {
+            id: tempDay.format(),
+            day: tempDay,
+          };
+        })
+    );
+  }
+  return calendar;
```

Comment:
Nice. Make sure you're keeping in mind how in the future you guys will want to use this calendar to place due dates. Right now it makes sense that as you iterate through the array you could compare the `id` with the due date of a task. What would you use the `day` field for?

Reply:
With the recent changes I made, I replaced the "day" field with a "number" field that represents the specific day's number. I guess the only purpose for it is to visually show the day number on the calendar. Thank you for all your suggestions.